packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -125,7 +125,7 @@ export class SwapQuoter {
         );
         const fastAbi = new FastABI(ERC20BridgeSamplerContract.ABI() as MethodAbi[]);
         const samplerContract = new ERC20BridgeSamplerContract(
-            this._contractAddresses.erc20BridgeSampler,
+            SAMPLER_ADDRESS,
```

Comment:
I do this because for other networks the `contracts-addresses` value is NULL_ADDRESS, which can screw with things (my things) expecting NULL_ADDRESS to not be a contract.